src/visualizations/index.js

```diff
@@ -1,2 +1,3 @@
 export { default as TimeSeries } from './time-series'
 export { default as Breakdown } from './breakdown'
+export { default as ScatterPie } from './scatter-pie'
```

Comment:
Please update `.eslintrc.js` with the following content and run `npm run lint -- --fix` to fix both new line at the end of file and dangling comma for multiline objects/arrays and alike:

```js
module.exports = {
  "env": {
    "browser": true,
    "es6": true,
  },
  "extends": [
    "eslint:recommended",
    "plugin:react/recommended",
  ],
  "parser": "babel-eslint",
  "parserOptions": {
    ecmaVersion: 6,
    "sourceType": "module",
  },
  "plugins": [
    "react",
  ],
  "rules": {
    "indent": [
      "error",
      2,
    ],
    "linebreak-style": [
      "error",
      "unix",
    ],
    "quotes": [
      "error",
      "single",
    ],
    "semi": [
      "error",
      "never",
    ],
    "no-console": [
      "warn",
    ],
    "comma-dangle": [
      "error",
      "always-multiline",
    ],
    "eol-last": [
      "error",
      "always",
    ],
  },
}
```